posthog-core/src/utils.ts

```diff
@@ -80,3 +165,31 @@ export function getFetch(): FetchLike | undefined {
 export const isFunction = function (f: any): f is (...args: any[]) => any {
   return typeof f === 'function'
 }
+
+// FNV-1a hash function
+// https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function
+// I know, I know, I'm rolling my own hash function, but I didn't want to take on
```

Comment:
Also, in this case, you're not actually rolling your own. You're just porting an implementation of a known function.